.dockerignore

```
@@ -0,0 +1,12 @@
+.env
+.env.j2
+.env.template
+.env.test
+.gitignore
+.github
+.vscode
+.pytest_chache
+.venv
+.gitpod.yml
+playbook.yml
+hosts.ini
```

One additional suggestions:
```suggestion
hosts.ini
**/__pycache__
```
Most mentioning because I don't expect people to necessarily be aware that `.dockerignore` has slightly different behaviour to `.gitignore`, in particular in that it _doesn't_ ignore subfolders by default - we need to explicitly reference them with the `**` prefix to mean arbitrary levels of nesting should be considered for this rule. I think that your existing entries wouldn't matters here, but this is one of those issues that can easily lead to weird confusion when some behaviour starts caching oddly!